File: completion-plugin/testData/csharp/IfStatement.cs
public class IfStatement
{
    public static void Main(string[] args)
    {
        if (DateTimeOffset.Now.ToUnixTimeMilliseconds() % 2 == 0)
        {
            System.Console.WriteLine("true");
        }
        else
        {
            System.Console.WriteLine("false");
        }
    }
}

File: completion-plugin/testData/java/IfStatement.java
public class IfStatement {
    public static void main(String[] args) {
        if (random() % 2 == 0) {
            test("true");
        } else {
            test("false");
        }
    }

    private void test(Object a) {
    }

    private int random() {
        return 3;
    }
}
